src/main/java/lotto/PurchaseAmount.java

```diff
@@ -0,0 +1,30 @@
+package lotto;
+
+public class PurchaseAmount {
+
+  private static final int PRICE_PER_LOTTO = 1000;
+  private static final String ERROR_INVALID_PURCHASE_AMOUNT =
+      "금액은 " + PRICE_PER_LOTTO + "원 단위로 입력해야 합니다.";
+
+  private final int value;
+
+  public PurchaseAmount(int value) {
+    validate(value);
+    this.value = value;
+  }
+
+  public int value() {
+    return value;
+  }
+
+  public int getTicketCount() {
```

Comment:
getTicketCount()와 같이 구현하면 인스턴스 변수로 int ticketCount 값을 가지고 있는 것으로 인식할 수 있다.
get을 쓰지 않고 아래와 같이 구현하는 것이 오히려 더 나은 의미를 부여할 수 있다.

```suggestion
  public int ticketCount() {
```

https://vimeo.com/1137582691 영상을 통해 메서드 이름 짓기 학습해볼 것을 추천